tests/kvmtest.sh

```diff
@@ -14,11 +14,11 @@ chmod 600 .ssh/id_rsa
 
 # Refresh dut in the virtual switch topology
 cd $SONIC_MGMT_DIR/ansible
-./testbed-cli.sh -m $inventory -t $testbed_file refresh-dut $tbname password.txt
+./testbed-cli.sh -m $inventory -t $testbed_file refresh-dut $tbname .password
```

Comment:
the test is broken, can you check?

Reply:
maybe that file does not exist on the test setup?

Reply:
1) can we rename the file on the setup so it is found and test work again?
or
2) I can change all ".password" to "password.txt" but that was generating more changes.
"28 times password.txt was used and 96 times .password in examples, made all docs use .password file for consistency of the commands and examples."
for me, either way, works. The fix was intended to bring consistency.